hack/release.sh

```diff
@@ -238,8 +241,9 @@ release_build() {
 	latest=
 	latestTgz=
 	if [ "$latestBase" ]; then
-		latest="$s3Dir/$latestBase"
-		latestTgz="$s3Dir/$latestBase.tgz"
+		latest="$s3Dir/$latestBase$binaryExt"
```

Comment:
we can remove/comment this line altogether because https://github.com/docker/docker/pull/21695/files#diff-cede583a27c227d458ca67079957c8a2R254 is commented out